package.json

```diff
@@ -45,17 +44,9 @@
     "auto": "^11.2.0",
     "better-docs": "2.7.3",
     "dotenv": "16.4.5",
-    "eslint": "^9.8.0",
-    "eslint-config-standard": "^17.1.0",
-    "eslint-plugin-import": "^2.29.1",
-    "eslint-plugin-jest": "28.6.0",
-    "eslint-plugin-n": "^17.10.1",
-    "eslint-plugin-node": "^11.0.0",
-    "eslint-plugin-promise": "^7.0.0",
     "jest": "^29.7.0",
     "jest-extended": "^4.0.2",
     "jsdoc": "4.0.3",
-    "progress": "^2.0.0",
```

Comment:
OK - this was added in https://github.com/noblox/noblox.js/commit/ababc28d13c9c798898730d3aa29677a7220715f for
https://github.com/noblox/noblox.js/commit/95c35b03714bc0a787b22814a6219f0994710f66 and is no longer needed